config/example/dbconfig.txt

```diff
@@ -1,28 +1,33 @@
-# MySQL Connection Configuration
+## MySQL Connection Configuration
+## This is used for stats, feedback gathering,
+## administration, and the in game library.
 
-# Server the MySQL database can be found at
+## Should SQL be enabled? Uncomment to enable.
+#SQL_ENABLED
+
+## Server the MySQL database can be found at.
 # Examples: localhost, 200.135.5.43, www.mysqldb.com, etc.
 ADDRESS localhost
 
-# MySQL server port (default is 3306)
+## MySQL server port (default is 3306).
 PORT 3306
 
-# Database the population, death, karma, etc. tables may be found in
-DATABASE paradise
-
-# Username/Login used to access the database
-LOGIN mylogin
+## Database for all SQL functions, not just feedback.
+FEEDBACK_DATABASE feedback
```

Comment:
I did update paradise_schema.sql to use feedback as well - it's what /tg/ uses, so I think it's best to have parity there.